source/04-MX/03.ptx

```diff
@@ -28,86 +28,159 @@ Consider the following linear system with a unique solution:
         </introduction>
         <task>
             <p>
-Define
```

Comment:
This activity (which is new to the 2023 edition and hadn't been taught before I think) was a bit of a dud and I ended up lecturing through it more or less. :-(